hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2501,4 +2501,15 @@
       The number of Recon Tasks that are waiting on updates from OM.
     </description>
   </property>
+  <property>
```

Comment:
I am going to commit this, but we have different way of writing configs. I missed it earlier. I will file a follow up JIRA to clean this up.